docs/docs/en/guide/task/conditions.md

```diff
@@ -51,7 +51,7 @@ After you finish creating the workflow, you can run the workflow online. You can
 
 In the above figure, the task status marked with a green check mark is the successfully executed task node.
 
-## Notice
+## Note
```

Comment:
Hi @sneh-wha I find some of the task doc change `notice` to `note` but some of them do not. Could you change the other part to keep them consistency